app/views/projects/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1><%= @project.name %></h1>
+<h3>Material: <%= @project.material %></h3><br>
+<h3>Challenge Theme: <%= @project.challenge.theme %></h3><br><br>
+<h3>Number of Contestants: <%= @project.contestants.count %><br>
```

Comment:
We don't want to do logic in our view. So you will want to create a method in your `Project` model that gets the count of contestants. 
```ruby
def num_contestants
 contestants.count
end
```
Then just call `@project.num_contestants` on line 4.